src/services/ui/src/pages/actions/setups/setupWithdrawPackage.ts

```diff
@@ -6,12 +6,28 @@ import {
 } from "@/pages/form/zod";
 
 export const medicaidWithdrawPackageSetup = {
-  schema: z.object({
-    additionalInformation: zAdditionalInfo,
-    attachments: z.object({
-      supportingDocumentation: zAttachmentOptional,
+  schema: z
+    .object({
+      additionalInformation: zAdditionalInfo.optional(),
+      attachments: z.object({
+        supportingDocumentation: zAttachmentOptional,
+      }),
+    })
+    .superRefine((val, ctx) => {
+      if (
+        !val.attachments.supportingDocumentation?.length &&
+        val.additionalInformation === undefined
+      ) {
+        ctx.addIssue({
+          message: "An Attachment or Additional Information is required.",
+          code: z.ZodIssueCode.custom,
+          fatal: true,
+        });
+        // Zod says this is to appease types
+        // https://github.com/colinhacks/zod?tab=readme-ov-file#type-refinements
+        return z.NEVER;
+      }
```

Comment:
The `superRefine` method adds an issue to zod's context via `ctx.addIssue`, and it [displays here](https://github.com/Enterprise-CMCS/macpro-mako/pull/322/files#diff-de3e84b14cfde777c5d8d16f66b2bb3a6bdab1b9b915d031e26f74eb9349a953R84-R98)